### DISCUSSION TITLE
[Projects beta] Allow "Issue" as a field type to create relationships between tickets

### DISCUSSION THREAD

**Discussion by TSampley** [community/community#16028] [Projects and Issues]
## Feature

I would like to be able to create fields of type "Issue" or at least create issue links with the "Text" field type.

## Context

I had setup two fields "Effort" and "Value" with ordering to help prioritize issues on a product backlog. I then noticed a couple of the high value items actually depended on the completion of other lower value items, and I thought it would be nice to filter or group by tasks with or without dependencies.

Like the other field types, the semantics of the field are up to the team, and I think the proposed "Issue" type is flexible enough to support more than just this one use-case.

**Comment by markbirbeck** [comment]
I'm going to piggy-back on this ticket... I would like the same feature in order to be able to add things like 'blocked by' or 'blocking' (as used in Kanban).

**Comment by SvenMeyer** [comment]
Same here : How can I create dependencies for tasks of a project ?

Google shows this sentence "With task lists you can break big issues into tasks, and then further organize your work with milestones and labels, and track relationships and dependencies."

... but it links to pages where I can not find any information on it.
